Podfile

```diff
@@ -1,8 +1,7 @@
-platform :osx, '10.9'
+platform :osx, '10.12'
```

Comment:
No, just trying to always have the latest libs and deprecation warnings (should we get any). I'll revert this for now